server/_dbConnect.ts

```diff
@@ -7,19 +7,21 @@ const host =
   process.env.NODE_ENV === 'development'
     ? 'localhost:8080'
     : process.env.MONGO_HOST
-const uri = `mongodb${prefix}://${root}:${pw}@${host}/?authSource=admin`
+const uri = `mongodb${prefix}://${root}:${pw}@${host}`
 
-const client = new MongoClient(uri)
+// stolen from https://github.com/vercel/next.js/tree/canary/examples/with-mongodb
```

Comment:
Yea it's under their example repository and you can actually create a repository from said template so I think they're fine with using it esp since we're linking to the original source